server/views/temporary-accommodation/assessments/index.njk

```diff
@@ -14,7 +14,7 @@
 
 {# TODO Move this HTML to the searchByCrnOrNameResult once HTML can be standardised between referrals and bookings #}
 {% set resultsHtml %}
-    {% if (not context.errors.prn) and ((tableRows|length > 0) or (not crn)) %}
+    {% if tableRows|length > 0 %}
```

Comment:
I believe we still need `or not crn` here, as otherwise we would show 'There are no results for ‘’ in ...' if there is nothing to show (e.g. no records for that view, at all). Possibly not the best UX in this case, but I think an empty table is less confusing than a message about an empty search the user haven't even submitted!